egs/librispeech/ASR/pruned_transducer_stateless/decode.py

```diff
@@ -250,6 +262,69 @@ def get_parser():
         Used only when --decoding_method is greedy_search""",
     )
 
+    parser.add_argument(
+        "--dynamic-chunk-training",
+        type=str2bool,
+        default=False,
+        help="""Whether to use dynamic_chunk_training, if you want a streaming
+        model, this requires to be True.
+        Note: not needed for decoding, adding it here to construct transducer model,
+              as we reuse the code in train.py.
+        """,
+    )
+
+    parser.add_argument(
+        "--short-chunk-size",
+        type=int,
+        default=25,
+        help="""Chunk length of dynamic training, the chunk size would be either
+        max sequence length of current batch or uniformly sampled from (1, short_chunk_size).
+        Note: not needed for decoding, adding it here to construct transducer model,
+              as we reuse the code in train.py.
+        """,
+    )
+
```

Comment:
You can move these options to train.py and import them into here.
See
https://github.com/k2-fsa/icefall/blob/dc89b61b808a22b64e7e38cb09771ea7a6bb64d2/egs/librispeech/ASR/pruned_transducer_stateless5/train.py#L331